ssh/tailssh/testcontainers/Dockerfile

```diff
@@ -3,6 +3,10 @@ FROM ${BASE}
 
 RUN groupadd -g 10000 groupone
 RUN groupadd -g 10001 grouptwo
-RUN useradd -g 10000 -G 10001 -u 10002 -m testuser
+RUN useradd -g 10000 -G 10001 -u 10002 testuser
```

Comment:
Removing the `-m` flag keeps this from creating a home directory for the user (we want PAM to do that on login).